fedocal/__init__.py

```diff
@@ -864,9 +870,30 @@ def add_meeting(calendar_name):
             day=form.meeting_date.data.day))
     elif flask.request.method == 'GET':
         form = forms.AddMeetingForm(timezone=tzone, calendars=calendars)
+        mtg_date = flask.request.args.get('date', None)
+        if mtg_date:
+            form.meeting_date.data = parser.parse(mtg_date).date()
+            form.meeting_date_end.data = parser.parse(mtg_date).date()
+
+        mtg_time = flask.request.args.get('time', None)
+        if mtg_time:
+            form.meeting_time_start.data = mtg_time
+            hours, minutes = mtg_time.split(':')[:2]
+            try:
+                hours = int(hours) + 1
+            except:
+                pass
```

Comment:
Heh.  Then about ``?time=01:aa``?

And, in this case, you let the ``"aa"`` pass through to the code below.  Should it instead be rejected or overwritten?